### PR TITLE
Disable login button until last onboarding slide

### LINKED ISSUE #166: Grey out Login on Onboarding til the last slide
# Why
We want to encourage people to read the onboarding text, and at the moment they'll be able to skip it.

### PR DESCRIPTION
**Pull request by maxgerber**
Resolves #166

### CODE REVIEW

File: server/views/layouts/main.hbs [hunk 1]
```diff
@@ -29,10 +29,17 @@
     flkty.on("change", function () {
       backgroundColours.forEach(function (colour) {
         splash.classList.remove("bg-" + colour)
-        login.classList.remove("bg-" + colour)
+        login.classList.remove("bg-" + colour, "o-20", "pointer", "grow", "disableButton")
       })
+
       splash.classList.add("bg-" + backgroundColours[flkty.selectedIndex])
       login.classList.add("bg-" + backgroundColours[flkty.selectedIndex])
+      console.log("flkty.selectedIndex", flkty.selectedIndex)
```
Comment (ivanmauricio) [inline server/views/layouts/main.hbs]: Rogue `console.log`

Comment (maxgerber) [reply server/views/layouts/main.hbs]: punctual!

Comment (maxgerber) [reply server/views/layouts/main.hbs]: Removed

**Comment by ivanmauricio** [comment]
@maxgerber just a `console.log` in there but apart from that ready to be merged!

**Pull request merged**